### PR TITLE
Main menus for both Windows and Android

### PR DESCRIPTION
**Pull request by ThatFlint**

### CODE REVIEW

**Comment by NilsHullegien** [comment]
Looks good for the initial main menu, we will obviously make this look better later on.

File: src/main/java/com/github/migi_1/Context/clientside/MainMenuAndroid.java [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+import de.lessvoid.nifty.builder.LayerBuilder;
+import de.lessvoid.nifty.builder.PanelBuilder;
+import de.lessvoid.nifty.builder.ScreenBuilder;
+import de.lessvoid.nifty.builder.TextBuilder;
+import de.lessvoid.nifty.controls.button.builder.ButtonBuilder;
+
+/**
+ *
+ * @author Remi
+ */
```
Comment (LMiljak) [inline src/main/java/com/github/migi_1/Context/clientside/MainMenuAndroid.java:21]: Missing Javadoc

File: src/main/java/com/github/migi_1/Context/clientside/MainMenuAndroid.java [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+    public static void main(String[] args) {
+        MainMenuAndroid app = new MainMenuAndroid();
+        app.start();
+    }
+    
+    /**
+     * Builds the menu.
+     */
+     @Override
+    public void simpleInitApp() {
```
Comment (LMiljak) [inline src/main/java/com/github/migi_1/Context/clientside/MainMenuAndroid.java:47]: This method is EXTREMELY long. Split it up!

File: src/main/java/com/github/migi_1/Context/clientside/MainMenuFunctionsAndroid.java [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+import com.jme3.app.state.AbstractAppState;
+import com.jme3.app.state.AppStateManager;
+import de.lessvoid.nifty.Nifty;
+import de.lessvoid.nifty.screen.Screen;
+import de.lessvoid.nifty.screen.ScreenController;
+
+/**
+ *
+ * @author Remi
+ */
```
Comment (LMiljak) [inline src/main/java/com/github/migi_1/Context/clientside/MainMenuFunctionsAndroid.java:18]: Missing Javadoc

File: src/main/java/com/github/migi_1/Context/serverside/MainMenu.java [hunk 4]
```diff
@@ -0,0 +1,194 @@
...
+import de.lessvoid.nifty.Nifty;
+import de.lessvoid.nifty.builder.ScreenBuilder;
+import de.lessvoid.nifty.builder.LayerBuilder;
+import de.lessvoid.nifty.builder.PanelBuilder;
+import de.lessvoid.nifty.builder.TextBuilder;
+import de.lessvoid.nifty.controls.button.builder.ButtonBuilder;
+/**
+ *
+ * @author Remi
+ */
```
Comment (LMiljak) [inline src/main/java/com/github/migi_1/Context/serverside/MainMenu.java:17]: Missing Javadoc

File: src/main/java/com/github/migi_1/Context/serverside/MainMenu.java [hunk 5]
```diff
@@ -0,0 +1,194 @@
...
+    public static void main(String[] args) {
+        MainMenu app = new MainMenu();
+        app.start();
+    }
+    
+    /**
+     * Builds the menu.
+     */
+     @Override
+    public void simpleInitApp() {
```
Comment (LMiljak) [inline src/main/java/com/github/migi_1/Context/serverside/MainMenu.java:43]: Method is EXTREMELY long, split it up.

File: src/main/java/com/github/migi_1/Context/serverside/MainMenuFunctions.java [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+import com.jme3.app.Application;
+import com.jme3.app.state.AbstractAppState;
+import com.jme3.app.state.AppStateManager;
+import de.lessvoid.nifty.Nifty;
+import de.lessvoid.nifty.screen.Screen;
+import de.lessvoid.nifty.screen.ScreenController;
+/**
+ *
+ * @author Remi
+ */
```
Comment (LMiljak) [inline src/main/java/com/github/migi_1/Context/serverside/MainMenuFunctions.java:17]: Missing Javadoc

File: src/main/java/com/github/migi_1/Context/serverside/MainMenuFunctions.java [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+    /**
+     * This function makes the program switch to a different screen.
+     * @param scr String
+     */
+    public void toScreen(String scr) {
+        nifty.gotoScreen(scr);
+    }
+    
+    /**
+     * This function quits the game
```
Comment (LMiljak) [inline src/main/java/com/github/migi_1/Context/serverside/MainMenuFunctions.java:84]: Missing dot at the Javadoc.

File: src/main/java/com/github/migi_1/Context/clientside/MainMenuFunctionsAndroid.java [hunk 8]
```diff
@@ -0,0 +1,86 @@
...
+    /**
+     * This function makes the program switch to a different screen.
+     * @param scr String
+     */
+    public void toScreen(String scr) {
+        nifty.gotoScreen(scr);
+    }
+    
+    /**
+     * This function quits the game
```
Comment (LMiljak) [inline src/main/java/com/github/migi_1/Context/clientside/MainMenuFunctionsAndroid.java:82]: Missing dot at the Javadoc.

**Comment by NilsHullegien** [comment]
Huge methods are split up now, comments from MrCrime were added.

**Comment by NilsHullegien** [comment]
No tests were added by the way, if these need to be added (might not be the case since this is GUI code) I can look at it.

**Pull request closed without merging**